TestUpdateConfiguration is flaky

Sometimes `TestUpdateConfiguration` fails.

```go
[2019-07-16T12:43:52.902Z]     
--- FAIL: TestUpdateConfiguration/ES_cluster_health_should_eventually_be_green (300.00s)
```
https://devops-ci.elastic.co/job/cloud-on-k8s-e2e-tests/111/

```go
[2019-07-15T09:24:16.084Z]     
--- FAIL: TestUpdateConfiguration/ES_cluster_health_should_eventually_be_green (300.00s)
```
https://devops-ci.elastic.co/job/cloud-on-k8s-e2e-tests/102/

--- a/operators/test/e2e/apm/configuration_test.go
+++ b/operators/test/e2e/apm/configuration_test.go
@@ -56,7 +56,7 @@ func TestUpdateConfiguration(t *testing.T) {
 
 	name := "test-apm-configuration"
 	esBuilder := elasticsearch.NewBuilder(name).
-		WithESMasterDataNodes(1, elasticsearch.DefaultResources)
+		WithESMasterDataNodes(3, elasticsearch.DefaultResources)
 	apmBuilder := apmserver.NewBuilder(name).
 		WithNamespace(test.Namespace).
 		WithVersion(test.ElasticStackVersion).

--- a/operators/test/e2e/es/failure_test.go
+++ b/operators/test/e2e/es/failure_test.go
@@ -25,7 +25,7 @@ import (
 
 func TestKillOneDataNode(t *testing.T) {
 	// 1 master + 2 data nodes
-	b := elasticsearch.NewBuilder("test-failure-kill-one-data-node").
+	b := elasticsearch.NewBuilder("test-failure-kill-a-data-node").
 		WithESMasterNodes(1, elasticsearch.DefaultResources).
 		WithESDataNodes(2, elasticsearch.DefaultResources)
 
@@ -40,7 +40,7 @@ func TestKillOneDataNode(t *testing.T) {
 
 func TestKillOneMasterNode(t *testing.T) {
 	// 2 master + 2 data nodes
-	b := elasticsearch.NewBuilder("test-failure-kill-one-master-node").
+	b := elasticsearch.NewBuilder("test-failure-kill-a-master-node").
 		WithESMasterNodes(2, elasticsearch.DefaultResources).
 		WithESDataNodes(2, elasticsearch.DefaultResources)
 

